<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-2LONNQF" ignored="1" name="Functional Regression Test" tests="1" failures="1" timestamp="14 Aug 2020 15:49:17 GMT" time="263.147" errors="0">
  <testcase name="LoginCheck" time="248.484" classname="com.qa.TestClass.TestClass">
    <failure type="java.lang.AssertionError" message="expected [ContactsNew] but found [Contacts]">
      <![CDATA[java.lang.AssertionError: expected [ContactsNew] but found [Contacts]
at com.qa.TestClass.TestClass.LoginCheck(TestClass.java:55)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- LoginCheck -->
  <testcase name="invalidDataLogin" time="0.0" classname="com.qa.TestClass.TestClass">
    <ignored/>
  </testcase> <!-- invalidDataLogin -->
</testsuite> <!-- Functional Regression Test -->
